Header header

bool acc_activation
bool aeb_activation
bool lcc_activation

bool acc_allowed
bool aeb_allowed
bool lcc_allowed

uint64 alive_rolling_counter_MABx
uint64 alive_rolling_counter_Jetson

float64 acc_speed_set_point
uint8 acc_gap_level
float64 veh_spd
float64 str_ang